plugins/doc_fragments/foreman_parameters.py

```diff
@@ -0,0 +1,59 @@
+# (c) 2019, Matthias Dellweg <dellweg@atix.de>
```

Comment:
you can have multiple fragments in one file, which I think we can use here?

https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-fragments

Reply:
TIL (and it's still quite early...)